lib/edge_impulse/ei_ncs.cmake

```diff
@@ -0,0 +1,3 @@
+zephyr_library_named(ei_ncs)
+zephyr_library_sources(ei_ncs.cpp)
+zephyr_library_link_libraries(edge_impulse)
```

Comment:
why is this in a `ei_ncs.cmake` file, and not directly in the CMakeLists.txt file ?

Reply:
I wanted to separate building the ei_ncs Zephyr library. I will move it to the CMakeLists.txt file.